get_facts_from_openstack_workaround.yml

```diff
@@ -0,0 +1,20 @@
+- name: Collect facts from Openstack instances - workaround
+  hosts: all
+  vars:
+  tasks:
+  - include_vars:
+      # file: "{{ lookup('file', '/var/tmp/openstack_creds') }}"
+      file: '/var/tmp/openstack_creds'
+      name: os_creds
+  - os_server_facts:
+      timeout: 60
+    environment:
+      OS_USERNAME: "{{ os_creds.clouds.osp11.auth.username }}"
+      OS_PASSWORD: "{{ os_creds.clouds.osp11.auth.password }}"
+      OS_AUTH_URL: "{{ os_creds.clouds.osp11.auth.auth_url }}"
+      OS_PROJECT_NAME:          "{{ os_creds.clouds.osp11.auth.project_name }}"
```

Comment:
I would like to fallow code spacing in playbooks as well. I know this for testing but good have.